packages/cfd/src/Containers/cfd-dashboard.tsx

```diff
@@ -249,6 +249,16 @@ const CFDDashboard = (props: TCFDDashboardProps) => {
         }
     });
 
+    React.useEffect(() => {
+        if (window.location.hash === '#real') {
+            setActiveIndex(0);
+        }
+        if (window.location.hash === '#demo-all' || window.location.hash === '#demo') {
+            setActiveIndex(1);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
> React Hook React.useEffect has an unnecessary dependency: 'window.location.hash'. Either exclude it or remove the dependency array. Outer scope values like 'window.location.hash' aren't valid dependencies because mutating them doesn't re-render the component.

The error message is very clear, I don't know why we always ignore `react-hooks/exhaustive-deps` instead of fixing it.

---

If you want to rerender to happen you should use `useLocation()` hook instead

```ts
const location = useLocation();

React.useEffect(() => {
    if (location.hash === '#real') {
        setActiveIndex(0);
    }
    if (location.hash === '#demo-all' || location.hash === '#demo') {
        setActiveIndex(1);
    }
}, [location.hash]);
```

Reply:
@farzin-deriv I think you are right, but in this file there are 2 more lines `'eslint-disable-next-line react-hooks/exhaustive-deps'` so I didn't think it's a big problem :) Ok, in a future I'll try to avoid this error. Also, thanks for advice about `useLocation()`, I've changed my PR